qiita_db/environment_manager.py

```diff
@@ -249,6 +249,9 @@ def make_environment(load_ontologies, download_reference, add_demo_user):
 def drop_environment(ask_for_confirmation):
     """Drops the database specified in the configuration
     """
+    # The transaction has an open connection to the database, so we need
+    # to close it in order to drop the environment
+    transaction.close()
```

Comment:
Since this is altering the system, it may be worth adding a note in the docstring.